twilio_mock.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'twilio_mock'
-  s.version     = '0.4.2'
+  s.version     = '0.5.1'
```

Comment:
why 0.5.1? it should be 0.5.0

Reply:
@MaicolBen been using this internally for now and just want to make sure we are using the correct version on our internal apps without relying on a commit hash.

Whenever I am done with the improvements, I'll address the issues on this PR & stick this gem to 0.5.0